### DISCUSSION TITLE
Names of dwellings in Russian localization

### DISCUSSION THREAD

**Discussion by DM-VLK** [ihhub/fheroes2#7594] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

The names of dwellings in the current localization, as in the Buka version, are different for a regular dwelling and an improved one. 

### Describe the possible solution

I propose to change them following the example of English localization, which, in my opinion, will allow you to quickly understand what type of dwelling has been built:

"Небесный замок" -> "Ул. Небесный замок"
"Мавзолей" -> "Ул. Мавзолей"
и.т.д.

**Comment by Branikolog** [comment]
Hello, @DM-VLK !
Current translation is based on "Buka" official localization. I'm not sure, we should change all that names, because players are got used to them after 20+ years passed. Also, in my personal opinion, your proposal changes original game aesthetics a bit. There would definitely be debates among players, which translation is better to leave, (for instance, `Небесный замок` or `Небесный чертог`). So, I really doubt it is really worth changing the original look.

**Comment by DM-VLK** [comment]
> players are got used to them after 20+ years passed

I understand you. My proposal was not about concrete translation, but about to leave only one name of the dwelling and to add the prefix "Ул." for upgraded dwellings like it did in the original version (Adobe/Upg. Adobe). It seems to me that Buka’s decision is unsuccessful, despite the fact that everyone is already used to it.

**Comment by Branikolog** [comment]
@DM-VLK 
Well, "Buka"-'s decision was kind of strange, but it's the official localization, which is 20+ year old. Each dwelling upgrade in game has a unique name, so we have to change quite a lot names. Not sure old players would accept changes. Also, as I mentioned before, we would have to decide, which of two dwelling names should be left in game, and this would cause hundreds of complains from players unsatisfied with the translation picked.

**Comment by DM-VLK** [comment]
@Branikolog 

Your point of view is clear, but maybe there is an opportunity to add an alternative translation or maybe make a poll on https://handbookhmm.ru? The Buka version appeared in 2003, and I met Heroes II in 1997. We had a very special pirated translation of the DOS version done by changing the font and it was translated like the English version.

![heroes2_000](https://user-images.githubusercontent.com/112470261/214921146-4cf6a2d9-7bff-406e-9beb-5b49d91a567a.png)

**Comment by Branikolog** [comment]
@DM-VLK 
I understand your points. 
We can initiate a poll somewhere at forums or even at our github channel, but I'm 99% sure, we would never come with unambiguous result. Heroes players are at the same time conservative and willing to change or mod everything. I can bet 2$ that the result of the poll will be 50/50 or so. )
You're the first person, actually, who was not satisfied with Buka translation (and mentioned it anywhere). :) You know, I'm the one, who speaks much to H2 and fheroes2 community. Most of players and streamers find it funny and accept these dwelling names without any hesitating usually. Once we have a mass feedback from people, regarding this translation, we would definitely change it. But for now I'm not sure, we need to open that "Pandora's box" of modifying original translation, since this would result in endless debates. ;) 
After all.... The original is the thing, everyone is undoubtedly considers.

**Comment by DM-VLK** [comment]
@Branikolog 

It's really not as important as how the game itself works. I'll try to start a poll in a few days and later give a link to the results.
In any case, the code is open and it will be possible to make changes in translation for myself.

**Comment by Branikolog** [comment]
@DM-VLK 
Sure!
It could be added even that funny pirated translation made by "Fargus", the code is open and everything is possible. But I personally think, there're lots of other problems, that contribution could be invested to. ;)

**Comment by ihhub** [comment]
Based on the above conversation I am moving this topic into the discussion section.
